### PR TITLE
Initial setup

### LINKED ISSUE #2: Docker Setup
### Description:
docker is cool

### Goal of the work:
set up Dockerfiles and docker-compose for development

### Acceptance Criteria:
docker-compose successfully launches containers for flask app, database, and api 
flask container returns 'hello world'
api container returns 'hello world'
database container shows healthy status

### Notes:
depending on choice of database, use postgres image to start

### PR DESCRIPTION
**Pull request by myshkins**
*revised docker-compose.yml
    * fixed volume mapping so that development servers can pickup changes and reload
    * added timezone environment variables
* added .gitignore
* removed unused directories

### CODE REVIEW

File: code/docker-compose.dev.yml [hunk 1]
```diff
@@ -35,6 +39,7 @@ services:
     environment:
       - POSTGRES_PASSWORD=password
```
Comment (hucksy) [inline code/docker-compose.dev.yml]: Maybe these should come from the .env file?

**Review by hucksy — CHANGES_REQUESTED** [review]
I dig it! I had a couple thoughts that maybe we can discuss on our call:
* Can we use a single .env file? seems a little hectic having them in multiple places
* Maybe we could add profiles to the docker compose file? these let you run only certain services for debugging/development: https://docs.docker.com/compose/profiles/
* Also do we need the top level  "code" directory?

**Comment by myshkins** [comment]
aight made those changes. I added the profiles, which is pretty slick thanks for pointing that out.

**Review by hucksy — APPROVED** [review]
righteous

**Pull request closed without merging**